src/memo.rs

```diff
@@ -85,19 +99,21 @@ pub struct Group {
     physical_plans: Vec<GroupPlanRef>,
     is_explored: bool,
     statistics: Option<Rc<Statistics>>,
+    lowest_cost_plans: HashMap<Rc<PhysicalProperties>, (Cost, GroupPlanRef)>,
```

Comment:
you can `type LowestCostPlans = HashMap<Rc<PhysicalProperties>, (Cost, GroupPlanRef)>;
` and use LowestCostPlans

Reply:
done